### PR TITLE
How to install Jenkins using Docker

### PR DESCRIPTION
**Pull request by Tumite**
I have read and understood the Contributor's Certificate of Origin available at the end of
https://raw.githubusercontent.com/hetzneronline/community-content/master/tutorial-template.md
and I hereby certify that I meet the contribution criteria described in it.
Signed-off-by: Roland Balla balla.roland96@gmail.com

### CODE REVIEW

File: tutorials/how-to-install-jenkins-using-docker/01.en.md [hunk 1]
```diff
@@ -0,0 +1,187 @@
+
```
Comment (n-se) [inline tutorials/how-to-install-jenkins-using-docker/01.en.md]: Please drop this line

File: tutorials/how-to-install-jenkins-using-docker/01.en.md [hunk 2]
```diff
@@ -0,0 +1,187 @@
...
+SPDX-License-Identifier: MIT
+path: "/tutorials/how-to-install-jenkins-using-docker"
+slug: "how-to-install-jenkins-using-docker"
+date: "2019-08-09"
+title: "How to install Jenkins using Docker"
+short_description: "This tutorial explains how to install Jenkins using Docker"
+tags: ["Linux", "Jenkins", "Ubuntu", "Docker"]
+author: "Roland Balla"
+author_link: "https://github.com/Tumite"
+author_img: "https://avatars3.githubusercontent.com/u/5360018?s=400&v=4"
```
Comment (n-se) [inline tutorials/how-to-install-jenkins-using-docker/01.en.md]:
```suggestion
author_img: "https://avatars3.githubusercontent.com/u/5360018"
```

File: tutorials/how-to-install-jenkins-using-docker/01.en.md [hunk 3]
```diff
@@ -0,0 +1,187 @@
...
+
+**Prerequisites**  
+
+To follow along with this tutorial, you will need an Ubuntu 18.04 server with:
+
+* at least 1GB of RAM
+
+* 10 GB+ of drive space
+
+* a `jenkins` user with sudo privileges
```
Comment (n-se) [inline tutorials/how-to-install-jenkins-using-docker/01.en.md]: This is not much work, can you please explain in short steps how to create one and how to set sudo permissions?

File: tutorials/how-to-install-jenkins-using-docker/01.en.md [hunk 4]
```diff
@@ -0,0 +1,187 @@
...
+
+## Step 1 - Installing Docker & Docker-Compose
+
+Docker is an application that simplifies the process of managing application processes in containers. Containers let you run your applications in resource-isolated processes.
+
+You can install up-to-date Docker by typing:
+
+```bash
+sudo curl -L get.docker.io | bash
```
Comment (n-se) [inline tutorials/how-to-install-jenkins-using-docker/01.en.md]: Better use an encrypted connection by default, so `curl -L https://get.docker.com`

File: tutorials/how-to-install-jenkins-using-docker/01.en.md [hunk 5]
```diff
@@ -0,0 +1,187 @@
...
+```
+
+You can check Docker version by running the following command:
+
+```bash
+sudo docker --version
+```
+
+`Optionally you can add jenkins user to the "docker" group will grant the ability to run whitout sudo, with usermod -aG sudo jenkins command`
```
Comment (n-se) [inline tutorials/how-to-install-jenkins-using-docker/01.en.md]: Why did you put the full sentence between `? You also did this a few times below

File: tutorials/how-to-install-jenkins-using-docker/01.en.md [hunk 6]
```diff
@@ -0,0 +1,187 @@
...
+
+On Linux, you can download the Docker Compose binary from the [Compose repository release page on GitHub](https://github.com/docker/compose/releases).
+
+Now we just typing:
+
+```bash
+sudo curl -L "https://github.com/docker/compose/releases/download/1.25.0/docker-compose-$(uname -s)-$(uname -m)" -o /usr/local/bin/docker-compose
```
Comment (n-se) [inline tutorials/how-to-install-jenkins-using-docker/01.en.md]: Please tell the user to check for new version first. I dont think it auto updates itself.

File: tutorials/how-to-install-jenkins-using-docker/01.en.md [hunk 7]
```diff
@@ -0,0 +1,187 @@
...
+
+Now you have a dockerized Jenkins. You only need to create you first Admin user in your browser and you can use it.
+  
+## Conclusion
+
+By following this tutorial you have successfully installed Jenkins with docker.
+
+##### License: MIT
+
+<!---
```
Comment (n-se) [inline tutorials/how-to-install-jenkins-using-docker/01.en.md]: Please drop the newlines so that it matches the original license from the template.

Comment (Tumite) [reply tutorials/how-to-install-jenkins-using-docker/01.en.md]: Sorry, i used stack-edit and maked for me a lot of newlines.

**Comment by Tumite** [comment]
Updated, check please. :)

File: tutorials/how-to-install-jenkins-using-docker/01.en.md [hunk 8]
```diff
@@ -0,0 +1,189 @@
...
+(c) The contribution was provided directly to me by some other person
+    who certified (a), (b) or (c) and I have not modified it.
+
+(d) I understand and agree that this project and the contribution are
+    public and that a record of the contribution (including all personal
+    information I submit with it, including my sign-off) is maintained
+    indefinitely and may be redistributed consistent with this project
+    or the license(s) involved.
+
+Signed-off-by: [submitter's name and email address here]
```
Comment (n-se) [inline tutorials/how-to-install-jenkins-using-docker/01.en.md]: Please sign off here

Comment (Tumite) [reply tutorials/how-to-install-jenkins-using-docker/01.en.md]: Updated.

**Comment by leojakeson** [comment]
@Tumite this is another tutorial that is very short and doesn't properly explain anything. These are not the kind of tutorials we are looking for (anymore), sorry.

If you open a new PR, please make sure the tutorial is at least two to three times as long as this one, and that it explains the steps being taken.

**Comment by Tumite** [comment]
Okay, on problem, i will explain better every step.

**Pull request closed without merging**